ompi/win/win.c

```diff
@@ -167,18 +167,18 @@ config_window(void *base, size_t size, int disp_unit,
                                      MPI_WIN_SIZE, size, true);
```

Comment:
"size" here isn't an MPI_Aint -- it's a size_t.  In most situations, they'll likely be the same -- but shouldn't this be cast to be 100% safe?